src/main/java/org/opentripplanner/middleware/models/MonitoredTrip.java

```diff
@@ -279,5 +302,68 @@ public boolean delete() {
         // TODO: Add journey state deletion.
         return Persistence.monitoredTrips.removeById(this.id);
     }
+
+    public List<NameValuePair> getParsedParams() throws URISyntaxException {
```

Comment:
I have been trying to see if the stuff plays well with the UI. This is what I understand is happening:
- `@BsonIgnore` seems to prevent reflection on the field/method on serialization, but reflection on the underlying types is still attempted on deserialization.
- `@JsonIgnore` seems to prevent reflection both on serialization and on deserialization. This is what we should use.

Reply:
Never mind that didn't seem to work. So this method should probably be renamed, e.g. `parseParams()`